modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -158,8 +158,10 @@ Cordova::
 +
 --
 // end::excludeDownstream[]
-Import and initialize Auth:
 
+. Import and initialize Auth.
+. Replace `import` with `require` in the `init` section, as displayed in the following snippet:
```

Comment:
don't think this line is needed as the snippet is already using `require` and not `import` e.g.
```
const Auth = require('@aerogear/auth').Auth;
```

Reply:
Remove this line
```
. Replace `import` with `require` in the `init` section, as displayed in the following snippet:
```

Reply:
Update made: https://github.com/aerogear/mobile-docs/pull/676/commits/77aef96c2f4ace32daf7bf4777cbfdd8c706824e